packages/services/f-http/src/tests/createClient.test.js

```diff
@@ -0,0 +1,42 @@
+import createClient from '../createClient';
+
+describe('createClient', () => {
+    describe('initialisation', () => {
+        it('should be defined', async () => {
+            // Arrange, Act & Assert
+            expect(createClient).toBeDefined();
+        });
+
+        it('should use default options when not overridden', async () => {
+            // Arrange & Act
+            const httpClient = createClient();
+            const mergedOptions = httpClient.readConfiguration();
+
+            // Assert
+            expect(mergedOptions).toBeDefined();
```

Comment:
You could shorten this to just checking the object itself? That way you will just have one expect?

Reply:
Yeah good idea, ill switch to this in the next version